### PR TITLE
Reset Google Analytics and remove unused files

### PR DESCRIPTION
**Pull request by xinyis991105**
## Description
This PR resets Google analytics for Autolab's documentation and removes some unused files in documentation as well.

## Motivation and Context

This PR addresses #1362. Removal of deprecated files is also important for future code maintenance.

## How Has This Been Tested?

mkdocs serve and the deprecation warning should not show up anymore. See material's [documentation](https://squidfunk.github.io/mkdocs-material/setup/setting-up-site-analytics/) for this alternative Google analytics setting. I decided to keep one-click.md since its content is not 100% covered by files currently in use. But if its content is no longer valuable, I can later remove it in this PR.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My change requires a change to the documentation, which is located at [Autolab Docs](https://github.com/autolab/docs)
- [ ] I have updated the documentation accordingly, included in this PR

## Other issues / help required

If unsure, feel free to submit first and we'll help you along.

### CODE REVIEW

**Comment by victorhuangwq** [comment]
I have looked into this and I am unable to confirm if the tracking is working. I ran the page locally, and assessed the site, but there were no users using it on Google Analytics

**Comment by xinyis991105** [comment]
May I ask how to test the functionality again? Do we need to re-deploy to see the effect of the change in mkdocs.yml?

**Comment by xinyis991105** [comment]
I rolled back to the older syntax with the strict mode off and deployed the GA4 property, replacing the UA property. Google Analytics is able to collect data now. My suggestion is to merge this PR into master and retain issue #1362. I doubt there is something funky going on with mkdocs-material codebase but any more efforts into this would be worthless at this point.

**Comment by victorhuangwq** [comment]
Hi Xinyi, I updated back to the new format of settings for mkdocs-material. I have deployed it and I have tested it to be working (you should test it too).

 I think I figured out why it wasn't working just now, it is likely because we were using an outdated version of mkdocs-material. 
 A simple `pip install --upgrade mkdocs-material` fixed it

**Comment by xinyis991105** [comment]
I have the latest version and mkdocs build does output the correct <script> tag for me this time. Thanks for helping out! Can you approve this PR so that we could close the issue as well?

File: mkdocs.yml [hunk 1]
```diff
@@ -51,13 +51,13 @@ extra:
...
-    - "auto"
+    analytics:
+        provider: google
+        property: G-PH5EYTKBNQ
 markdown_extensions:
     - admonition
     - codehilite
       #linenums: true
-strict: true
+strict: false
```
Comment (victorhuangwq) [inline mkdocs.yml:61]: I think I forgot to change this back to true. Could you change it to true and I will approve it?

**Review by victorhuangwq — CHANGES_REQUESTED** [review]

File: mkdocs.yml (hunk 1 above)
Comment (xinyis991105) [reply mkdocs.yml]: Done.

**Comment by victorhuangwq** [comment]
Does deploying it work?

**Review by victorhuangwq — APPROVED** [review]
LGTM. Deploying works

**Pull request merged**